runtime/autoload/provider.vim

```diff
@@ -0,0 +1,18 @@
+" Common functionality for providers
+
+let s:stderr = {}
+
+function! provider#stderr_collector(chan_id, data, event) dict
```

Comment:
Should this be a dict function?
I've seen `E725: Calling dict function without Dictionary: provider#stderr_collector` - not sure if it coming from the python provider or clipboard though.